github/actions_permissions_enterprise.go

```diff
@@ -25,7 +25,7 @@ type ActionsPermissionsEnterprise struct {
 	SelectedActionsURL   *string `json:"selected_actions_url,omitempty"`
 }
 
-func (a ActionsPermissionsEnterprise) String() string {
```

Comment:
I don't think this change is correct.  There may be more of these.  I'll let you find them.

Reply:
Fixed, thanks